src/prommis/solvent_extraction/sx_model_flowsheet_dynamic.py

```diff
@@ -0,0 +1,186 @@
+""" 
+Demonstration flowsheet for dynamic state solvent extraction loading process
+using parameters and data derived from West Kentucky No. 13 coal refuse.
+
+Authors: Arkoprabho Dasgupta
+
+"""
+
+from pyomo.environ import ConcreteModel, SolverFactory, units, TransformationFactory
+
+from idaes.core import FlowDirection, FlowsheetBlock
+from idaes.core.initialization import InitializationStatus
+from idaes.core.initialization.block_triangularization import (
+    BlockTriangularizationInitializer,
+)
+
+from prommis.leaching.leach_solution_properties import LeachSolutionParameters
+from prommis.solvent_extraction.ree_og_distribution import REESolExOgParameters
+from prommis.solvent_extraction.solvent_extraction import SolventExtraction
+
+"""
+Method of building a dynamic solvent extraction model with a specified number of 
+stages and with two separate property packages for the two inlet streams.
+This is a loading operation, so no additional argument has to be specified.
+
+"""
+
+m = ConcreteModel()
```

Comment:
Could you put all this code into sub-methods, and add a `if __name__ == "__main__":` at the end. See the UKy or leaching flowsheet for an example. This makes it a lot easier to test and run the code.

Reply:
I will check it

Reply:
Could you please address this issue?

Reply:
I had applied this change some time back, there was an issue with code coverage in Git, so I had rewritten it. Andrew did not say anything about this after that.